time stamp
uint32 image_height
uint32 image_width
uvdar_core/Point2DWithFloat[] points
